### PR TITLE
makefile for rfc processing

### PR DESCRIPTION
**Pull request by dericed**

### CODE REVIEW

File: rfc_frontmatter.markdown [hunk 1]
```diff
@@ -0,0 +1,15 @@
+% Title = "Extensible Binary Meta Language"
+% abbrev = "EBML"
+% category = "info"
```
Comment (retokromer) [inline rfc_frontmatter.markdown]: Should be:
`category = "std"`

File: rfc_frontmatter.markdown [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+% docName = ""
```
Comment (retokromer) [inline rfc_frontmatter.markdown]: You should give some input, e.g.:
`docName = "draft-lhomme-cellar-ebml-00"`

File: rfc_frontmatter.markdown [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+% ipr= "trust200902"
+% area = ""
```
Comment (retokromer) [inline rfc_frontmatter.markdown]: I guess should be:
`area = "art"`

**Comment by mbunkus** [comment]
Thanks.

**Pull request merged**